modules/ibi-installer-installation-config.adoc

```diff
@@ -34,7 +34,13 @@ If the seed image requires a separate private registry authentication, add the a
 |Specification|Type|Description
 |`shutdown`|`boolean`|Specifies if the host shuts down after the installation process completes. The default value is `false`.
 |`extraPartitionStart`|`string`|Specifies the start of the extra partition used for `/var/lib/containers`. The default value is `-40Gb`, which means that the partition will be exactly 40Gb in size and uses the space 40Gb from the end of the disk. If you specify a positive value, the partition will start at that position of the disk and extend to the end of the disk.
-|`extraPartitionLabel`|`string`|The label of the extra partition you use for `/var/lib/containers`. The default label is `varlibcontainers`.
+|`extraPartitionLabel`|`string`|The label of the extra partition you use for `/var/lib/containers`. 
+
+[NOTE]
+====
+The default partition label is `varlibcontainers`. If you set a different partition label, you must ensure it matches the partition label set in the seed image. For more information, see "Configuring a shared container partition between ostree stateroots".
```

Comment:
@tsorya is this true?, I think the current default is var-lib-containers - hence it doesn't match what we document in [Configuring a shared container partition between ostree stateroots](https://85062--ocpdocs-pr.netlify.app/openshift-enterprise/latest/edge_computing/image_base_install/ibi-preparing-for-image-based-install#cnf-image-based-upgrade-shared-container-partition_ibi-preparing-image-based-install)

Reply:
current default is varlibcontainers , you can it here https://github.com/openshift/installer/blob/master/pkg/asset/imagebased/image/imagebased_config.go#L33

Reply:
I'm confused!
If IBI use `varlibcontainers`
and IBU use `varlibcontainers` - [here](https://docs.openshift.com/container-platform/4.17/edge_computing/image_based_upgrade/preparing_for_image_based_upgrade/cnf-image-based-upgrade-shared-container-partition.html)

Where do we have `var-lib-containers`?